src/Unleash/Unleash.csproj

```diff
@@ -71,6 +71,7 @@
     </PackageReference>
     <PackageReference Include="murmurhash" Version="1.0.3" />
     <PackageReference Include="NuGet.Versioning" Version="6.1.0" />
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
```

Comment:
Force a reference to System.Text.Json for older versions that don't have it built in